spec/helpers/models_helper.rb

```diff
@@ -0,0 +1,34 @@
+def create_and_activate_user(name)
+  user = User.create!(
+    name:, email: "#{name}@example.com", password: '123456', bio: "Bio of #{name}", photo: "#{name}.jpg"
+  )
+  user.confirm
+  user.save
+  user
+end
+
+def create_posts_for_user(user, count: 1)
+  posts = []
+  count.times do |i|
+    posts << Post.create!(
+      user:,
+      title: "Post #{i}",
+      text: "This is body of post #{i}",
+      comments_counter: 0,
+      likes_counter: 0
+    )
+  end
+  posts
+end
+
+def create_comments_for_post_by_user(post, user, count: 1)
+  comments = []
+  count.times do |i|
+    comments << Comment.create!(
+      user:,
+      post:,
+      text: "Comment #{i}"
+    )
+  end
+  comments
+end
```

Comment:
- [ ] Hi @ahangarha, your tests are not running successfully. Let's fix it. Screenshot:

![Screenshot from 2022-06-15 23-06-29](https://user-images.githubusercontent.com/63087491/173940774-c42c095b-4056-4e56-8c42-9fa890bdb381.png)

Reply:
Are you sure you run the app with proper gem versions?
We didn't have any issue running this code on our systems

Reply:
@ahangarha, I think the issue arises when you omitted those values. You are creating a user without the name field.

Reply:
I am following linter and also the test passes completely on my system.